internal/pkg/policy/monitor.go

```diff
@@ -306,7 +306,7 @@ func (m *monitorT) updatePolicy(pp *ParsedPolicy) bool {
 		Logger()
 
 	if newPolicy.CoordinatorIdx <= 0 {
-		zlog.Info().Msg("ignore policy that has not pass through coordinator")
+		zlog.Info().Str(logger.PolicyId, newPolicy.PolicyId).Msg("ignore policy that has not passed through coordinator")
```

Comment:
```suggestion
		zlog.Info().Str(logger.PolicyId, newPolicy.PolicyId).Msg("Ignore policy that has not passed through coordinator")
 ```

Reply:
Thanks for the fix, applied it directly locally to my diff.